plugins/modules/network_settings_intent.py

```diff
@@ -2000,6 +2108,8 @@ def main():
         if state != "deleted":
             dnac_network.get_want(config).check_return_status()
         dnac_network.get_diff_state_apply[state](config).check_return_status()
+        if config_verify:
+            dnac_network.validate_dnac_config(state, config).check_return_status()
```

Comment:
In our last meeting, we discussed this to write like:
```
 dnac_network.get_diff_state_apply[state](config).check_return_status()
        if config_verify:
            dnac_network.verify_diff_state_apply[state](config).check_return_status()
```
Then write seperate APIs for merged, deleted state as like get_diff_state_apply....

```
self.verify_diff_state_apply = {
                                     'merged': self.verify_diff_merged,
                                     'deleted': self.verify_diff_deleted,
                                     'replaced': self.verify_diff_replaced,
                                     'overridden': self.verify_diff_overridden,
                                     'gathered': self.verify_diff_gathered,
                                     'rendered': self.verify_diff_rendered,
                                     'parsed': self.verify_diff_parsed
                                     }

```
and then write all the base class APIs. Then override the required APIs in derived class.